ROOTFS/media/rpi/sage-utils/dhcp-pxe/nfs/etc/systemd/timesyncd.conf.d/waggle-fallback-ntp.conf

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
I would drop the Nvidia specific comments and add our own that state this file was derived from an NVidia file or something.

Reply:
done.